pkg/csi/plugins/nodeserver.go

```diff
@@ -192,9 +192,11 @@ func (ns *nodeServer) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 	if os.IsNotExist(err) {
 		glog.V(3).Infof("NodeUnpublishVolume: targetPath %s has been cleaned up, so it doesn't need to be unmounted", targetPath)
 		return &csi.NodeUnpublishVolumeResponse{}, nil
-	}
-	if err != nil {
-		return nil, errors.Wrapf(err, "NodeUnpublishVolume: stat targetPath %s error %v", targetPath, err)
+	} else if err != nil {
+		// if error is "transport endpoint is not connected", ingore and do umount
+		if !errors.Is(err, syscall.ENOTCONN) {
+			return nil, errors.Wrapf(err, "NodeUnpublishVolume: stat targetPath %s error %v", targetPath, err)
```

Comment:
Please also provide the log info for syscall.ENOTCONN.

Reply:
The error will be logged in the code below: https://github.com/fluid-cloudnative/fluid/blob/52499ccf5b923159bfda66652e2f58998500afb6/pkg/csi/plugins/nodeserver.go#L204-L206